app/assets/stylesheets/custom.css

```diff
@@ -0,0 +1,12 @@
+.logout {
+  margin-right: 10px;
+}
+
+.leitbild-text{
```

Comment:
had to add this custom CSS for the text to sit in the middle of the images

@sidewinder2020 just incase you want to change the picture descriptions, this is where you'll find that custom css and class. The class is leitbild-text on the index.html.erb of the home